hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/snapshot/OMSnapshotCreateRequest.java

```diff
@@ -300,7 +300,7 @@ private void removeSnapshotInfoFromSnapshotChainManager(
   ) {
     try {
       snapshotChainManager.deleteSnapshot(info);
-    } catch (IllegalStateException exception) {
+    } catch (IOException exception) {
```

Comment:
Do you want to remove this now that ISE is changed to IOException? 
https://github.com/apache/ozone/blob/f8c42fa86724d5de0ee6511b2af8fb444c4611f4/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/snapshot/OMSnapshotCreateRequest.java#L194

Reply:
Ah, I missed it. Removed.

Thanks for the catch.